E3_Distill_ESRGAN/libs/models/student_rrdb.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, first_call=True):
 
   def call(self, inputs):
     intermediates = [inputs]
+    residual_junction = tf.math.add_n(intermediates)
```

Comment:
Not sure if that makes sense architecturally. `intermediates` is a *growing* list that collects all the tensors from the outputs of the intermediate residual blocks.
In other words, the values of variables `residual_junction` and `intermediates` updates in every iteration of the loop. So, I think they should be kept there.